run_elrond_tests.py

```diff
@@ -82,7 +85,7 @@ def config_to_kast_term(config):
 # 10 = 8 zeros for all SC addresses + 2 zeros as placeholder for the VM type.
 sc_addr_reserved_prefix_len = sc_addr_num_leading_zeros + 2
 
-sys.setrecursionlimit(1500000000)
+sys.setrecursionlimit(2100000000)
```

Comment:
Just curious: does this really need a recursion limit this high? Would such a recursive call fit in memory? I think that most things I ran for Elrond + WASM used something like 4000...